<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartSettingsDialog</class>
 <widget class="QDialog" name="StartSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>151</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>151</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Краулер</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>272</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>272</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Адрес сайта:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="addressLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Пауза между запросами к сайту в миллисекундах:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="timePauseSpinbox">
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>3000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Максимальное количество ссылок для обхода сайта:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>0 в качестве максимального количества ссылок является бесконечным обходом</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>311</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="maxCrawlUrlsSpinbox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
